docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
@bexelbie I am not sure about the `restart` in `vagrant service-manager restart cockpit` . Basically we are asking user to restart the service even if under the hood it is starting it.